components/aws/sagemaker/tests/unit_tests/run_all_tests.sh

```diff
@@ -1,4 +1,4 @@
 export PYTHONPATH=../../
 
-coverage run -m pytest --ignore=tests/test_utils.py --junitxml ./unit_tests.log
+coverage run -m pytest --junitxml ./unit_tests.log
```

Comment:
remind me why were we ignoring this earlier ?

Reply:
it was used as "utils for tests"
i think it would be better to name that file "utils.py" and keep it at root of unit tests folder 

now its being used as to test common/utils.py functions